src/java.base/share/classes/java/util/zip/ZipEntry.java

```diff
@@ -702,4 +710,20 @@ public Object clone() {
             throw new InternalError(e);
         }
     }
+
+    /**
+     * Validate that the CEN header size + name length + comment length + extra length
+     *  do not exceed 65,535 bytes per the PKWare APP.NOTE
+     *  4.4.10, 4.4.11, & 4.4.12.
+     * @param name Zip entry name
+     * @param extra Zip extra data
+     * @param comment Zip entry comment
+     * @return true if valid CEN Header size; false otherwise
+     */
+    private static  boolean isCENHeaderValid(String name, byte[] extra, String comment) {
```

Comment:
Extra space before `boolean`

```suggestion
    private static boolean isCENHeaderValid(String name, byte[] extra, String comment) {
```

Reply:
fixed